scenarios/zero_arc_create_and_read/src/main.rs

```diff
@@ -0,0 +1,177 @@
+use anyhow::anyhow;
+use holochain_types::prelude::Record;
+use holochain_types::prelude::{ActionHash, Timestamp};
+use holochain_wind_tunnel_runner::prelude::*;
+use holochain_wind_tunnel_runner::scenario_happ_path;
+use std::collections::HashSet;
+use std::time::SystemTime;
+use timed_integrity::TimedEntry;
+
+#[derive(Debug, Default)]
+struct ScenarioValues {
+    sent_actions: u32,
+    seen_actions: HashSet<ActionHash>,
+}
+
+impl UserValuesConstraint for ScenarioValues {}
+
+fn agent_setup(
+    ctx: &mut AgentContext<HolochainRunnerContext, HolochainAgentContext<ScenarioValues>>,
+) -> HookResult {
+    if ctx.assigned_behaviour() == "zero_read" || ctx.assigned_behaviour() == "zero_write" {
+        ctx.get_mut()
+            .holochain_config_mut()
+            .with_target_arc_factor(0);
+    }
+    start_conductor_and_configure_urls(ctx)?;
+    install_app(ctx, scenario_happ_path!("timed"), &"timed".to_string())?;
+    if ctx.assigned_behaviour() == "zero_read" || ctx.assigned_behaviour() == "zero_write" {
+        try_wait_for_full_arc(ctx)?;
```

Comment:
I see this was added in the prior PR, but I now realize I find the name confusing -- maybe it could be changed to make clearer that we're waiting to see another peer with a full arc, not that we're waiting for ourselves to claim a full arc.

Reply:
Okay, renamed to `try_wait_until_full_arc_peer_discovered`.